### ISSUE TITLE
As far as this script can detect, your system is not configured to work with League's anticheat.

### ISSUE THREAD

**Issue by flakesrc** [lutris/lutris#2467]
**Describe the bug**

EDIT: 
Now the behavior is almost the same, the game appears the symbol but does not open, but now the error windows do not appear before this happens.

I get these two windows with messages when I try to open lol:

- It looks like you already configured your system to work with League anticheat, and saved the setting to persist across reboots. However, for some reason the persistence part did not work.

For now, would you like to enable the setting again until the next reboot? <<< clicking "yes" nothing happens

- As far as this script can detect, your system is not configured to work with League's anticheat. Please verify that you can get into the practice too before playing a multiplayer game.

And  i have: 
```
sudo cat /etc/sysctl.conf          
abi.vsyscall32 = 0
abi.vsyscall32 = 0
```

**Expected behavior**

the game open normally

**Current behavior**

Game does not open because of description messages

**Steps to reproduce**

1. Open Lutris
2. Click in "play" on lol game

**Lutris debugging output**

https://pastebin.com/saBsX6fD

**System information**

this not works but i have this:
```
Kernel: 5.3.11-arch1-1 
wine-4.19
```
GPU: RX 570 MSI OC DUAL
CPU: Ryzen 5 3600
Gpu driver: mesa/amdgpu

**Screenshots**

Video: https://vimeo.com/373474762

**Comment by tjackson** [comment]
Arch deprecated the location at /etc/sysctl.conf. You want that file to be at /etc/sysctl.d/99-sysctl.conf. You can check if running "sudo sysctl -p /etc/sysctl.conf works first. /etc/sysctl.conf can be a symlink to /etc/sysctl.d/99-sysctl.conf. too.

See here: https://wiki.archlinux.org/index.php/Sysctl#Configuration

**Comment by flakesrc** [comment]
```
➜  sysctl.d sudo sysctl -p /etc/sysctl.conf            
abi.vsyscall32 = 0
➜  sysctl.d sudo sysctl -p /etc/sysctl.d/99-sysctl.conf
abi.vsyscall32 = 0
```

**Comment by flakesrc** [comment]
I have the same problem after reboot

**Comment by tjackson** [comment]
terminate called after throwing an instance of 'dxvk::DxvkError'

I think you're missing a driver or Vulkan is not working? But maybe you can look here: https://github.com/Joshua-Ashton/d9vk/issues/260

**Comment by flakesrc** [comment]
oh..., now game open, I just need to see if it will update and open normally, using the other version, I could log in, but then upgrade to the new interface and then not open again, in a few hours I comment again here

**Comment by flakesrc** [comment]
PS: this fucking lib use 100% cpu  of my ryzen 5 3600 and 7gb ram

**Comment by flakesrc** [comment]
Now the game    not open after update (the same problem with  the other installer version)
video: https://vimeo.com/373495775
last outputs:
```
Running /home/flakesrc/.local/share/lutris/runners/wine/lutris-lol-4.17-x86_64/bin/wine /home/flakesrc/Games/league-of-legends/drive_c/Riot Games/League of Legends/LeagueClient.exe
Running /home/flakesrc/Games/league-of-legends/syscall_check.sh
Waiting on children
Exit with returncode 0
DEBUG    2019-11-15 18:42:08,968 [command.on_stop:161]:The process has terminated with code 0
DEBUG    2019-11-15 18:42:08,968 [command.stop:246]:Management process looks dead already.
DEBUG    2019-11-15 18:42:08,968 [command.stop:258]:logger already detached
LeagueClient.exe(00000009): ALWAYS| Application Version:9.22.297.3335 - CL:2973335 - Build Date:Nov 11 2019 - Build Time:15:12:54
LeagueClient.exe(00000009): ALWAYS| Direct Launch enabled for league_of_legends.live in environment 'live' for app path 'C:\Riot Games\League of Legends\LeagueClient.exe' and install dir 'C:\Riot Games\League of Legends'
LeagueClient.exe(00000009): ALWAYS| Found associated Riot Client install (C:/Riot Games/Riot Client/RiotClientServices.exe)
LeagueClient.exe(00000009): ALWAYS| Riot Client started. Exiting with the expectation of being restarted (ExitForDirectLaunch).
Waiting on children
Waiting on children
```

**Comment by flakesrc** [comment]
help on new issue please: https://github.com/lutris/lutris/issues/2468
